### DISCUSSION TITLE
FIrefly Autobudgeting - Cron appears successful but no changes made

### DISCUSSION THREAD

**Discussion by Stupv** [firefly-iii/firefly-iii#8217] [Firefly III]
Morning Team,

I checked in today (01/12 in my timezone) to find that Firefly hasnt updated the autobudgets (first month rollover since i migrated from Docker to Native for the same reason and configured the cronjob). I checked and found my system timezone was incorrectly set, so it was still 22:54 on 30/11. I fixed that on the OS, then manually triggered the cronjob with:

/usr/bin/php /var/www/html/firefly-iii/artisan firefly-iii:cron

Which reported it was successful on all fronts, however when i open up Firefly it still thinks its november (the 'default view' is november not december) and the autobudgets havent updated for the new month.

Any leads onw here i should be looking?

On a side note - since moving from docker to native, when i use the importer i have to reauthenticate it every single time. It's a minor inconvenience but since i'm making a thread thought i'd ask about it

**Comment by Stupv** [comment]
Everything rolled over on the 2nd, which is weird but fine

**Reply by JC5** [reply]
Check what the time is according to the container. Perhaps the timezone is not set. You can do this on the /debug page.

**Comment by Stupv** [comment]
Hi JC,

It's running natively not in docker

**Reply by JC5** [reply]
Same question. Are you sure the timezone is correct? The cron job is programmed to trigger on the first of the month.

Set debug mode if you want to find out more (next month):

https://docs.firefly-iii.org/firefly-iii/faq/other/#how-do-i-enable-debug-mode
